app/src/androidTest/assets/test.html

```diff
@@ -28,13 +28,10 @@ <h2>Service worker</h2>
 <script type="text/javascript">
         // Add a key value pair to local storage. This is required by WebViewDataTest which asserts
         // that the local storage directory should exist
```

Comment:
Can we add a comment saying that this test is for now geckoview only?

Reply:
@npark-mozilla the test should still work with WebView :) Or is it GV only because of that assertion?

Reply:
Wait, I think I'm misunderstanding.  Thought GV doesn't have service workers in private mode, so I thought you removed them, but it is appended. How does it work exactly now?  (I thought it won't work on WV because service worker.js is disabled)

Reply:
service workers were disabled on GV with privacy mode. Switched to just loading the JS dynamically with Javascript which gives us similar behavior for the tests 👍